main.c

```diff
@@ -31,19 +32,29 @@ void main(void)
 	__enable_irq();
 
 	// MUST CALL THIS FUNCTION BEFORE MODIFYING YEAR, MONTH, DAY, ETC...
-	initRTC();
+	//initRTC();
 
 	// TODO HAVE USER SET CURRENT YEAR,MONTH,DAY,TIME
 
+<<<<<<< HEAD
+	// MAKE SURE TO CALL THIS FUNCTION AFTER INITIAL TIME INPUT
+	//startRTC();
+
+   // SCB->SCR |= SCB_SCR_SLEEPONEXIT_Msk;
+    initUART();
+    initUARTWriteOnly();
+=======
 	// MAKE SURE TO CALL THIS FUNCTION WHEN WE WANT TO START KEEPING TRACK OF TIME
 	// startRTC();
 
     InitI2C(ACCEL_ADDRESS);
+>>>>>>> master
 
-    SCB->SCR |= SCB_SCR_SLEEPONEXIT_Msk;
 
     while (1)
     {

```

Comment:
Just in general, main should be as simple as possible right now because it's what we all use. That's why we'll just have the Inits and sleep function.